app/helpers/index.js

```diff
@@ -0,0 +1,36 @@
+const bcrypt = require('bcryptjs');
+const { encryptionError } = require('../errors');
+const saltRounds = 10;
```

Comment:
Move this to a constant file

Reply:
Do you mean adding it to the configuration file or simply to create a file to hold all project constants?

Reply:
Yes, a constant file in this case it's fine, a configuration in this case seems like a overkill